### DISCUSSION TITLE
Moving Quasar project to new PC

### DISCUSSION THREAD

**Discussion by kendatom** [quasarframework/quasar#13584] [General - Components / Directives / etc] [answered]
Hello,
are there any instructions on how to move existing project to a new PC. Which folders are necessary to copy and how to initialize the project?
Thank you.

**Comment by MilosPaunovic** [comment, marked as answer]
Copy everything except `node_modules` and just run `npm i` or `yarn` on the new machine.

**Reply by kendatom** [reply]
Thank you, that was quick.
